docs/admin/extensible-admission-controllers.md

```diff
@@ -126,12 +126,16 @@ initializers:
           - pods
 ```
 
+After you create the `initializerConfiguration`, the system will take a few
+seconds to honor the new configuration. Then, `"podimage.example.com"` will be
+appended to the `metadata.initializers.pending` field of newly created pods. You
+should already have a ready "podimage" initializer controller that handles pods
+whose `metadata.initializers.pending[0].name="podimage.example.com"`. Otherwise
+the pods will stuck uninitialized.
+
 Make sure that all expansions of the `<apiGroup, apiVersions, resources>` tuple
```

Comment:
I left it here because explaining what this InitializerConfiguration does seemed more important. Can move if necessary, but I think it reads fine.